Cargo.lock

```diff
@@ -4991,7 +5004,7 @@ version = "21.5.0"
 dependencies = [
  "async-trait",
  "bollard",
- "byteorder",
+ "byteorder 1.5.0",
```

Comment:
This means that there are now two versions of byteorder in use. That might be unnecessary.

Reply:
nice catch!